dev/com.ibm.ws.jaxws.2.3.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,26 @@
+###############################################################################
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+###############################################################################
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME com.ibm.ws.build.example.project/resources/OSGI-INF/l10n/metatype.properties
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+
+webserviceClient=JAX-WS Web Service Client Configuration
+webserviceClient.desc=Web Service configuration for client.
+
+serviceName=Web Service Client Service Name
+serviceName.desc=The serviceName defined in the @WebServiceClient annotation
```

Comment:
Web server client service name
The serviceName property defined in the @WebServiceClient annotation.

Reply:
I shouldn't have had two instances of service in this sentence, but `web server` doesn't apply to the terminology, so I've changed it to `Web service client name`